### PR TITLE
[UI smoke test] For #6531: ads UI test for close tab button

### PR DESCRIPTION
**Pull request by sv-ohorvath**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### CODE REVIEW

File: app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt:42]: I'm OK with adding this back if we think it won't cause any Firebase issues with short timeouts. I figure if we wait longer on Firebase there's less chance of timing problems, that was my thinking when I removed it in the refactor - I was wondering the benefit. I'll leave it up to you.

**Review by AaronMT — APPROVED** [review]
Wait on landing for feedback on from my question.

File: app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt
Comment (sv-ohorvath) [reply app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt:42]: I thought we can use this when we don't expect a long wait as for page loading, if some view doesn't show in 5 sec, maybe it's not worth waiting longer thus increasing the total time of the test run. But I don't mind following your advice, as you think is better.

**Pull request merged**
